src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java

```diff
@@ -128,6 +129,15 @@
         return new ResponseEntity<>(errorMessage, HttpStatus.FORBIDDEN);
     }
 
+    @ExceptionHandler(EntityAmountException.class)
+    public ResponseEntity<ErrorMessage> handleEntityAmountException(EntityAmountException e,
+                                                                    ServletWebRequest request) {
+        ErrorMessage errorMessage = getErrorMessage(request, HttpStatus.NOT_ACCEPTABLE,
+                "exception.max-amount-of-children", Collections.singletonList(e.getLocalizedMessage()));
+        logErrorMessage(WARN, errorMessage);
+        return new ResponseEntity<>(errorMessage, HttpStatus.NOT_ACCEPTABLE);
+    }
+
```

Comment:
I think you also shoud test this handling in `GlobalExceptionHandlerTest`

Reply:
Ok. you are right